src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -62,32 +62,38 @@
 				</div>
 
 				<!-- Loan tag -->
-				<loan-tag-v2 v-if="showTags && !isLoading" :loan="loan" :amount-left="amountLeft" />
+				<router-link
+					:to="customLoanDetails ? '' : `/lend/${loanId}`"
+					v-kv-track-event="['Lending', 'click-Read more', 'Photo', loanId]"
+					class="tw-flex hover:tw-no-underline"
+				>
+					<loan-tag-v2 v-if="showTags && !isLoading" :loan="loan" :amount-left="amountLeft" />
```

Comment:
The loan tag will show an underline sometimes when clicked due to `focus`. I don't think we have `focus:tw-no-underline` available, so the parent `router-link` probably needs a new class and an entry added to the styles:

```
.loan-card-tag:focus {
  @apply tw-no-underline;
}
```

Reply:
Looks like `focus:tw-no-underline` is working fine